packages/lab/src/overlay/Overlay.css

```
@@ -1,93 +1,46 @@
-/* Experimental - Overlay css variable API */
 .saltOverlay {
-  --overlay-text-color: var(--salt-content-primary-foreground);
-  --overlay-spacing: var(--salt-size-unit);
-  --overlay-fontSize: var(--salt-text-fontSize);
-  --overlay-background: var(--salt-container-primary-background);
-  --overlay-borderColor: var(--salt-container-primary-borderColor);
-  /* TODO: z-index audit, provide z index here or rely on popper */
-  --overlay-zindex: var(--salt-zIndex-flyover);
+  --overlay-borderColor: var(--saltOverlay-borderColor, var(--salt-container-primary-borderColor));
+  --overlay-background: var(--saltOverlay-background, var(--salt-container-primary-background));
 }
 
-.saltOverlay-secondary.saltOverlay {
-  --overlay-background: var(--salt-container-secondary-background);
-  --overlay-borderColor: var(--salt-container-secondary-borderColor);
-}
+.saltOverlay {
+  font-family: var(--salt-text-fontFamily);
+  font-size: var(--saltOverlay-fontSize, var(--salt-text-fontSize));
+  font-weight: var(--saltOverlay-fontWeight, var(--salt-text-fontWeight));
+  line-height: var(--saltOverlay-lineHeight, var(--salt-text-lineHeight));
 
-.saltOverlay-content {
-  display: flex;
-  padding: calc(var(--overlay-spacing) * 2);
-  font-size: var(--overlay-fontSize);
+  border-color: var(--overlay-borderColor);
+  border-style: var(--saltOverlay-borderStyle, var(--salt-container-borderStyle));
+  border-width: var(--saltOverlay-borderWidth, var(--salt-size-border));
 
-  border: 1px solid var(--overlay-borderColor);
-  box-shadow: var(--salt-overlayable-shadow-popout);
-  line-height: var(--salt-text-lineHeight);
   background: var(--overlay-background);
-  color: var(--overlay-text-color);
-  position: relative;
-  opacity: 1;
-  overflow: visible;
-  z-index: var(--overlay-zindex);
-}
+  box-shadow: var(--saltOverlay-boxShadow, var(--salt-overlayable-shadow-popout));
```

shadow should be modal
```suggestion
  box-shadow: var(--saltOverlay-boxShadow, var(--salt-overlayable-shadow-modal));
```

---

@bhoppers2008  has confirmed, the correct token is `overlayable-shadow-popout`, figma needs updating